GPflow/transforms.py

```diff
@@ -236,12 +277,12 @@ def tf_forward(self, x):
         return tf.squeeze(fwd) if self.squeeze else fwd
 
     def tf_log_jacobian(self, x):
```

Comment:
I think it's best not to hard-code `tf.float64`, rather use `settings.dtypes.float_type`